src/requests/features.jl

```diff
@@ -59,6 +55,16 @@ function get_definitions(x::EXPR, tls::StaticLint.Scope, server, locations)
     end
 end
 
+safe_isfile(s) = false
+function safe_isfile(s::AbstractString)
+    try
+        !occursin("\0", s) && isfile(s)
+    catch err
+        isa(err, Base.IOError) || isa(err, Base.SystemError) || rethrow()
```

Comment:
Do we really need to rethrow anything here? I don't really see any cases where we can fix anything that causes `isfile` to fail, right?

Reply:
I think we want to see what error conditions might occur in crash reporting, so I generally think we should only "mask" exceptions that we understand and then rethrow anything else so that we see it in crash reporting and can then make a decision whether we also want to mask it, or whether there is some other root cause for it.